File: mpp1/mpp1/Model/Incident.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace mpp1.Model;

public class Incident
{
    [Key]
    public Guid Id { get; set; }
    [JsonIgnore]
    public virtual Vehicle? Vehicle { get; set; }
    public virtual Guid VehicleId { get; set; }
    public string Location { get; set; }
    public string Description { get; set; }
    [Range(0,Int32.MaxValue)]
    public int Cost { get; set; }
    public DateTime WhenHappend { get; set; }
}

File: scripts/databaseScripts/model/Incident.py
from Utilities import get_string_in_single_quotes

class Incident():
    def __init__(self, id, vehicle_id, location, description, cost, when_happend):
        self.__id = id
        self.__vehicle_id = vehicle_id
        self.__location = location
        self.__description = description
        self.__cost = cost
        self.__when_happend = when_happend

    def __str__(self) -> str:
        return f"({get_string_in_single_quotes(self.__id)}," + \
               f"{get_string_in_single_quotes(self.__vehicle_id)}," + \
               f"{get_string_in_single_quotes(self.__location)}," + \
               f"{get_string_in_single_quotes(self.__description)}," + \
               f"{self.__cost}," + \
               f"{get_string_in_single_quotes(self.__when_happend)})" 